pkg/plugins/manager/client/client.go

```diff
@@ -232,7 +231,10 @@ func (s *Service) attachJWT(ctx context.Context, pluginCtx backend.PluginContext
 		return ctx
 	}
 
-	token, err := s.jwtAuthService.Generate(store.GetUserIDString(user), pluginCtx.PluginID)
+	// TODO: pull tenant from context
+	tenantID := 0
```

Comment:
@jalevin IIUC, we will pull this from environment variables, correct?

Reply:
I believe that is correct. cc: @stephaniehingtgen confirm?

Reply:
Yup! It'll be an env variable `HG_STACK_ID`